css_def/def.go

```diff
@@ -0,0 +1,48 @@
+package css_def
+
+import (
+	"encoding/json"
+
+	"sourcegraph.com/sourcegraph/srclib/graph"
+)
+
+func init() {
+	graph.RegisterMakeDefFormatter("Dir", newDefFormatter)
+}
+
+// DefData should be kept in sync with the def 'Data' field emitted by the
+// basic grapher.
+type DefData struct {
+	Name      string
```

Comment:
Good Catch @alexsaveliev! - I've [updated](https://github.com/sourcegraph/srclib-css/pull/2/commits/255164e2a51e348b19c9f0981b1689a30b80932d) `srclib-css` to include `Data` used by the [default](https://github.com/sourcegraph/sourcegraph/pull/1906/commits/cfb9a10be5fce783bee43685e18d0a9d8db3e71d) formatter.

Addressed on: https://github.com/sourcegraph/srclib-css/pull/2/commits/255164e2a51e348b19c9f0981b1689a30b80932d